modules/fundamental/tests/weakness/mod.rs

```diff
@@ -0,0 +1,97 @@
+#![allow(clippy::expect_used)]
+
+use roxmltree::Document;
+use std::io::Read;
+use test_context::test_context;
+use test_log::test;
+use trustify_common::{hashing::HashingRead, model::Paginated};
+use trustify_entity::labels::Labels;
+use trustify_module_fundamental::weakness::service::WeaknessService;
+use trustify_module_ingestor::{graph::Graph, service::weakness::CweCatalogLoader};
+use trustify_test_context::{document_read, TrustifyContext};
+use zip::ZipArchive;
+
+#[test_context(TrustifyContext)]
+#[test(tokio::test)]
+async fn simple(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
+    let graph = Graph::new(ctx.db.clone());
+    let loader = CweCatalogLoader::new(&graph);
+    let service = WeaknessService::new(ctx.db.clone());
+
+    // extract document from zip file
+
+    let zip = document_read("cwec_latest.xml.zip").await?;
+
+    let mut archive = ZipArchive::new(zip)?;
+
+    let entry = archive.by_index(0)?;
+
+    let mut hashing = HashingRead::new(entry);
+    let mut xml = String::new();
+    hashing.read_to_string(&mut xml)?;
+    let digests = hashing.finish()?;
+
+    // load
+
+    let doc = Document::parse(&xml)?;
+    loader.load(Labels::default(), &doc, &digests).await?;
+
+    // fetch data
+
+    let all = service
+        .list_weaknesses(
+            Default::default(),
+            Paginated {
+                offset: 0,
+                limit: 10,
+            },
+        )
+        .await?;
+
+    assert_eq!(964, all.total);
```

Comment:
nit to replace the magic number with a const here and on line 83

`const TOTAL_ITEMS_FOUND:u64 = 964;`

Reply:
Done.